gcc4mbed/external/mbed/libraries/mbed/targets/hal/TARGET_NXP/TARGET_LPC43XX/analogin_api.c

```diff
@@ -25,110 +28,145 @@
 #define ANALOGIN_MEDIAN_FILTER      1
 
 static inline int div_round_up(int x, int y) {
-  return (x + (y - 1)) / y;
+	return (x + (y - 1)) / y;
```

Comment:
Are you changing spaces to tabs or something here? I see multiple places in the diff where the indentation has been modified for no reason by these commits.